### ISSUE TITLE
Implement "Papers" page

### ISSUE THREAD

**Issue by ghost** [ocaml/v3.ocaml.org-rescript#129]
Similar in structure to this component, but with categories of papers instead of categories of news.

![image](https://user-images.githubusercontent.com/1385802/113063294-12f93f80-9183-11eb-951c-689c183a5dda.png)

Subtasks:
- [x] Generalize the "other news stories" component" so that it can be used here (this will be discussed further in this issue #348)
- [x] Define content structure for papers database and populate with sample data

**Comment by ghost** [comment]
>  Define content structure for papers database and populate with sample data

This was done in ood recently.

**Comment by ghost** [comment]
> Generalize the "other news stories" component" 

This will be merged into master soon once PR #393 is complete.

**Comment by ghost** [comment]
The remaining task is to use the "highlights in quadrants" component with four categories of papers. The page has not been designed in Figma yet, so just pick some arbitrary categories and icons to start, and then create a follow-up issue to make the page match the Figma design when it emerges.

**Comment by ghost** [comment]
@ILeandersson Do you already have a design for this page?

**Comment by ghost** [comment]
@rdavison If we abstract the layout of the News page into a page template, then this page could be the second instance of that template.
